apps/server/src/interfaces/remote-app.gateway.ts

```diff
@@ -0,0 +1,143 @@
+import { OnGatewayConnection, OnGatewayInit, WebSocketGateway, WebSocketServer } from '@nestjs/websockets';
+import { Inject, Logger } from '@nestjs/common';
+import { ConfigType } from '@nestjs/config';
+import { Server, WebSocket } from 'ws';
+import serverConfig from '../config/server.config';
+import { NetworkService } from '../utilities/network.service';
+import { protocolV0 } from '@flybywiresim/remote-bridge-types';
+
+type ClientType = 'aircraft' | 'remote';
+
+interface RemoteBridgeConnection {
+  type: ClientType;
+  clientName: string;
+  clientID: string;
+}
+
+@WebSocketGateway({
+  cors: { origin: '*' },
+  path: '/interfaces/v1/remote-app',
+})
+export class RemoteAppGateway implements OnGatewayInit, OnGatewayConnection {
+  constructor(
+    @Inject(serverConfig.KEY) private serverConf: ConfigType<typeof serverConfig>,
+    private networkService: NetworkService,
+  ) {}
+
+  private readonly logger = new Logger(RemoteAppGateway.name);
+
+  @WebSocketServer() server: Server;
+
+  async afterInit(server: Server) {
+    this.server = server;
+    this.logger.log('Remote app gateway websocket initialised');
+    this.logger.log(
+      `Initialised on http://${await this.networkService.getLocalIp(true)}:${this.serverConf.port}${server.path}`,
```

Comment:
```suggestion
      `Initialised gateway on ws://${await this.networkService.getLocalIp(true)}:${this.serverConf.port}${server.path}`,
```

I believe that a user looking at the logs might think this is (also) the remote app path and try to access it but of course it returns 404 since its only for websockets, and might cause confusion.